static/js/src/advantage/react/components/Subscriptions/ReBuyExpired/ReBuyExpiredButton/ReBuyExpiredButton.tsx

```diff
@@ -86,7 +86,12 @@ const BuyButton = ({
         userInfo?.customerInfo?.email
       )}`;
     } else {
-      location.pathname = "/pro";
```

Comment:
BTW, this was actually a bug fix: after concluding a purchase, I was getting redirected to /pro, which was very confusing.